transformations/bq/sql/_DROPS.sql

```diff
@@ -0,0 +1,7 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+DROP SCHEMA IF EXISTS `@@BQ_PROJECTID@@.transformation`;
```

Comment:
`@@BQ_PROJECTID@@.transformations`? Or simply `@@BQ_PROJECTID@@.@@BQ_DATASET_TRANSFORMATIONS@@`

Reply:
I set it has @@BQ_PROJECTID@@.transformation as we previously had this module called `transformation`, and it's deployed in both staging and production. As we want to drop the previous one that's why I don't had the `s` or `@@BQ_DATASET_TRANSFORMATIONS@@`

Reply:
Ok! I imagined something like that.